moto/dynamodb2/models.py

```diff
@@ -409,6 +409,15 @@ def add(self, old, new):
         seq = len(self.items) + self.starting_sequence_number
         self.items.append(
             StreamRecord(self.table, t, event_name, old, new, seq))
+        result = None
+        from moto.awslambda import lambda_backends
```

Comment:
Any reason to import this inline vs at the module level?

Reply:
I've followed the style in the sqs/models.py, where a lambda function is kicked off whenever an SQS message is received. There the lambda_backends  is also imported locally.
I'm ambivalent where the import happens really - it shouldn't be too resource-heavy.

Reply:
Ah, no. This is right. Sorry about that.